configurations/coding_plans.py

```diff
@@ -19,8 +19,8 @@ def clean_age_with_range_filter(text):
         return Codes.NOT_CODED
 
 
-def get_rqa_coding_plans(pipeline_name):
-    if pipeline_name == "dadaab_pipeline":
+def get_rqa_coding_plans(season_name):
+    if season_name == "s01_dadaab":
```

Comment:
What's the advantage of adding the complexity of another configuration field for the season vs. encoding it in the pipeline name e.g. PipelineName == "wusc-keep-ii-kakuma-s01"?

Reply:
encoding it in the pipeline name much simpler option.Have removed season references.